subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -153,16 +153,21 @@ static void handle_wifi_twt_event(struct net_mgmt_event_callback *cb)
 	const struct wifi_twt_params *resp =
 		(const struct wifi_twt_params *)cb->info;
 
-	print(context.sh, SHELL_NORMAL, "TWT response: %s for dialog: %d and flow: %d\n",
-	      wifi_twt_setup_cmd2str[resp->setup_cmd], resp->dialog_token, resp->flow_id);
-
-	/* If accepted, then no need to print TWT params */
-	if (resp->setup_cmd != WIFI_TWT_SETUP_CMD_ACCEPT) {
-		print(context.sh, SHELL_NORMAL,
-		      "TWT parameters: trigger: %s wake_interval_ms: %d, interval_ms: %d\n",
-		      resp->setup.trigger ? "trigger" : "no_trigger",
-		      resp->setup.twt_wake_interval_ms,
-		      resp->setup.twt_interval_ms);
+	if (resp->resp_status == WIFI_TWT_RESP_NOT_RECEIVED) {
+		print(context.sh, SHELL_NORMAL, "TWT response status: %s\n",
```

Comment:
This doesn't make sense, response status but response not received? we need to fix terminology b/w RPU status and TWT response status. Something like:

if (rpu_stats == WIFI_TWT_RESP_NOT_RECEIVED)
   print ("TWT response timed out")